partiql-planner/src/main/kotlin/org/partiql/planner/builder/PartiQLPlannerBuilder.kt

```diff
@@ -18,7 +18,7 @@ import org.partiql.planner.internal.SqlPlanner
  */
 public class PartiQLPlannerBuilder {
 
-    private val flags: MutableSet<PlannerFlag> = mutableSetOf()
+    private val flags: MutableSet<PlannerFlag> = mutableSetOf(PlannerFlag.REPLACE_WITH_REFS)
```

Comment:
self-review:  by default, preserve prior behavior to rewrite all WITH variable references to the query representation. using the below `replaceWithReferences` with a value of false, will not rewrite the variable references